virtcontainers/qemu.go

```diff
@@ -914,6 +917,41 @@ func (q *qemu) hotplugAddMemory(memDev *memoryDevice) error {
 	return q.sandbox.storage.storeHypervisorState(q.sandbox.id, q.state)
 }
 
+func (q *qemu) hotplugVSock(op operation) (uint32, error) {
+	if op == removeDevice {
+		return 0, errors.New("Not implemented")
+	}
+
+	return q.hotplugAddVSock()
+}
+
+func (q *qemu) hotplugAddVSock() (uint32, error) {
+	// setup qmp channel if necessary
+	if q.qmpMonitorCh.qmp == nil {
+		qmp, err := q.qmpSetup()
+		if err != nil {
+			return 0, err
+		}
+
+		q.qmpMonitorCh.qmp = qmp
```

Comment:
I see that we aren't consistent on the QMP channel setup throughout the file, and that this appears to be boiler plate code.  Should we standardize?  See hotplugAddMemory and hotplugBlockDevice.  WDYT?

Reply:
yes, I've seen the same with in the muitples places we setup a QMP channel.  I agree we should refactor that to avoid code duplication.

Reply:
Or even as a first step, at least be consistent, since we seem to have a couple different patterns here...